.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -1,40 +1,62 @@
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+<!--
+Please fill out ALL required sections. Bug reports with missing information will be
+closed.
 
-**To reproduce**
-Steps to reproduce the behavior:
+Before submitting a bug report:
+
+- Check if the bug has already been fixed by updating WordPress and/or Gutenberg.
+- Check if the bug is caused by a plugin by deactivating all plugins except Gutenberg.
+- Check if the bug is caused by a theme by activating a default theme e.g. Twenty Twenty.
+
+If this is a security issue, please report it in HackerOne instead:
+https://hackerone.com/wordpress
+-->
+
+## Description
+<!-- Please write a brief description of the bug. -->
+
+## Step-by-step reproduction instructions
+<!--
+Please list the steps needed to reproduce the bug. For example:
 1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
-
-**Expected behavior**
-A clear and concise description of what you expected to happen.
-
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
-
-**Editor version (please complete the following information):**
-- WordPress version: [e.g: 5.3.2]
-- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default? [e.g: "gutenberg plugin", "default"]
-- If the Gutenberg plugin is installed, which version is it? [e.g., 7.6]
-
-**Desktop (please complete the following information):**
- - OS: [e.g. iOS]
- - Browser [e.g. chrome, safari]
- - Version [e.g. 22]
-
-**Smartphone (please complete the following information):**
- - Device: [e.g. iPhone6]
- - OS: [e.g. iOS8.1]
- - Browser [e.g. stock browser, safari]
- - Version [e.g. 22]
-
-**Additional context**
-- To report a security issue, please visit the WordPress HackerOne program: https://hackerone.com/wordpress.
+2. Click on '...'
+3. Scroll down to '...'
+-->
+
+## Expected behaviour
+<!-- Please describe what you expected to happen. -->
+
+## Actual behaviour
+<!-- Please describe what actually happened. -->
+
+## Screenshots or screen recording (optional)
+<!--
+If possible, please upload a screenshot or screen recording which demonstrates
+the bug. You can use LIEcap to create a GIF screen recording:
+https://www.cockos.com/licecap/
+-->
+
+## Plugin code (optional)
+<!--
+If this bug is to do with a developer API, please share a code snippet or plugin
+code that demonstrates the issue. You can use GitHub Gist to share multiple code
+files:
+https://gist.github.com
+-->
+
+## WordPress information
+- WordPress version: <!-- e.g. "5.6.0" -->
+- Gutenberg version: <!-- e.g. "9.4.0" or "Not installed" -->
+- Are all plugins except Gutenberg deactivated? <!-- "Yes" or "No" -->
+- Are you using a default theme (e.g. Twenty Twenty)? <!-- "Yes" or "No" -->
+
+## Device information
+- Device: <!-- e.g. "Desktop" or "iPhone 11" -->
+- Operating system: <!-- e.g. "Windows 10" or "iOS 14" -->
+- Browser: <!-- e.g. "Chrome 86.0" or "Mobile Safari" -->
```

Comment:
It might be a WordPress mobile app as well.  @hypest and @Tug, do you report bugs in this repository? I guess you do 😃

Reply:
Yeap, we do report native mobile issues in this repo too 👍

Maybe we can change this line to something like `Client: <!-- e.g. "Chrome 86.0" or "Mobile Safari" or "WordPress Android" or "WordPress iOS" -->`?

Reply:
I noticed we have a seperate bug report template for the mobile apps. Could we consolidate into one?

Reply:
> Could we consolidate into one?

Yes, that makes sense, thanks Robert! Consolidating in this very PR could do it I think.

While at it, may I suggest introducing a "Step 0. On the <client>" on the steps to reproduce, to state which client the steps are about? That might help with staying in the right context while reading the steps. WDYT?

Reply:
I decided not to consolidate the two templates because having them separated means that GitHub can automatically add the `Mobile App Android/iOS` label which I assume is desirable.

I did update the mobile bug report template based on our changes to the web bug report template, through.

Reply:
Good point about the label adding. Keeping them separate can still work and we can always revise if need arises. Thanks!